airqo-platform/AirQo-api
# Publishing mqtt bridge

- What does this PR do?
This PR created new endpoints for publishing telemetry events and carrying out configurations on both the mqtt and http protocol bridges.

- What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/jira/software/projects/APA/boards/5?selectedIssue=APA-71).

- How do I test out this PR?
Git clone this repo `cd AirQo-api/src/device-registry` 
Then run `npm install` to get all the necessary dependencies 
Afterwards run `npm run test` to test out the end points using any REST client tool available to you.

- Which end points should are ready for testing?
partial url: /api/devices/mqtt/config/gcp [GET]
partial url: /api/devices/http/config/gcp [GET]
partial url: /api/devices/mqtt/config/gcp [PUT]
partial url: /api/devices/http/config/gcp [PUT]
partial url: /api/devices/mqtt/publish/gcp [POST]
partial url: /api/devices/http/publish/gcp [POST]
